Client/multiplayer_sa/CMultiplayerSA_WidescreenFix.cpp

```diff
@@ -0,0 +1,1111 @@
+/*****************************************************************************
+ *
+ *  PROJECT:     Multi Theft Auto
+ *  LICENSE:     See LICENSE in the top level directory
+ *  FILE:        multiplayer_sa/CMultiplayerSA_WidescreenFix.cpp
+ *
+ *  Multi Theft Auto is available from https://www.multitheftauto.com/
+ *
+ *****************************************************************************/
+
+#include "StdInc.h"
+
+struct RsGlobal
+{
+    char* AppName;
+    int   width;
+    int   height;
+    int   frameLimit;
+    int   quit;
+    int   ps;
+};
+
+#ifndef M_PI
+    #define M_PI 3.14159265358979323846
+#endif
+
+float fFrontendWidth[41], fFrontendHeight[41];
+float fMiscWidth[30], fMiscHeight[30];
+float fRadarWidth[23], fRadarHeight[23];
+float fHUDWidth[120], fHUDHeight[120];
+float fCameraWidth[2], fCameraHeight[2];
+
+float fWideScreenWidthScale = 1.0f;
+float fWideScreenHeightScale = 1.0f;
+
+static const float fHudHeightScale = 1.0f;
+static const float fHudWidthScale = 1.0f;
+static const float fRadarWidthScale = 1.0f;
+static const float fRadarHeightScale = 1.0f;
```

Comment:
constexpr will be better imo

Reply:
changed and removed static

Reply:
Static here makes sense, so don't remove it. Just change const to constexpr

Reply:
I just put it back. Could you say more about `static`? it's lifecycle thing, I guess, but I don't get it

Reply:
The static keyword restricts the visibility of a variable or function to the source file and ensures that the variable is allocated at program start and remains in memory for the program's entire duration

Reply:
thanks